### PR TITLE
Use Symfony/Uuid instead of Ramsey/Uuid

### PR DESCRIPTION
**Pull request by brusch**

### CODE REVIEW

File: models/Tool/UUID.php [hunk 1]
```diff
@@ -138,7 +138,11 @@ public function createUuid()
             throw new \Exception('No instance identifier specified.');
         }
 
-        $this->uuid = (string) \Ramsey\Uuid\Uuid::uuid5(\Ramsey\Uuid\Uuid::NAMESPACE_DNS, $this->getInstanceIdentifier() . '~' . $this->getType() . '~' . $this->getItemId())->toString();
+        // namespace originally used from \Ramsey\Uuid\Uuid::NAMESPACE_DNS
+        $namespace = \Symfony\Component\Uid\Uuid::fromString('6ba7b810-9dad-11d1-80b4-00c04fd430c8');
+        $uuid = \Symfony\Component\Uid\Uuid::v5($namespace, $this->getInstanceIdentifier() . '~' . $this->getType() . '~' . $this->getItemId());
```
Comment (aarongerig) [inline models/Tool/UUID.php:143]:
```php
$uuid = \Symfony\Component\Uid\Uuid::v5(\Symfony\Component\Uid\Uuid::NAMESPACE_DNS, ...);
```

see https://github.com/ramsey/uuid/blob/master/src/Uuid.php#L50
Also, consider not using the fully qualified name, instead cleanup with `use` statements for better readability.

e.g.:
```php
use Symfony\Component\Uid\Uuid;
...
$uuid = Uuid::v5(Uuid::NAMESPACE_DNS, ...);
```

**Comment by dpfaffenbauer** [comment]
👍

**Comment by brusch** [comment]
@dpfaffenbauer thanks again 😉

File: models/Tool/UUID.php (hunk 1 above)
Comment (brusch) [reply models/Tool/UUID.php:143]: I think PhpStorm did this automatically, since the class itself is also named `UUID`. 😉

Comment (aarongerig) [reply models/Tool/UUID.php:143]: Oh, I see... nevermind then. ;)

Comment (brusch) [reply models/Tool/UUID.php:143]: Just changed it manually in 23c898e27661eb7889fd97ebfc41d41d864a3e04 
Thanks!

Comment (jdreesen) [reply models/Tool/UUID.php:143]: I just created #8224 for using the DNS namespace constant instead of the magic string.

**Pull request merged**